kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/documentation/links/NameAndUrlPopoverViewImpl.html

```diff
@@ -0,0 +1,35 @@
+<!--
+  ~ Copyright 2019 Red Hat, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<div data-field="popover" data-placement="top" data-toggle="popover" data-trigger="manual" data-container="#popover-container" title="" data-close="true" xmlns="http://www.w3.org/1999/html">
```

Comment:
I think we can remove some attributes here:
```suggestion
<div data-field="popover" data-placement="top" data-toggle="popover" data-trigger="manual" data-container="#popover-container" data-close="true">
```